README.md

```diff
@@ -4,7 +4,7 @@
 
 # flame_example
 
-This is set of small examples showcasing specific features of the Flame Engine, so it's a great source of learning for how to do certain things.
+This is set of small examples showcasing specific features of the Flame Engine, so it's a great source of learning for how to do certain things. [See it live here](https://flame-engine.github.io/flame_example).
```

Comment:
Maybe we should add this to our domain instead of using the GitHub.io url? I think we can just config a subdomain for this, like examples.flame-engine.org wdyt?

Reply:
yeah I like that

Reply:
I am gonna merge this, we can do that as a follow up